app/crawler/fetcher.py

```diff
@@ -0,0 +1,29 @@
+from selenium import webdriver
+import requests
+
```

Comment:
Great concept! I got the idea but might need some more examples to get a feeling for how and when to use it...